crawler/config.py

```diff
@@ -9,6 +9,36 @@ class RSSConfig:
     links = {
         "The Verge": "https://www.theverge.com/rss/index.xml",
         "Engadget": "https://www.engadget.com/rss.xml",
+        "Life Hacker": "https://lifehacker.com/rss",
+        "Wired": "https://www.wired.com/feed/rss",
         # Feedburner based XML Feed, need to be careful
```

Comment:
Careful how? Please state exactly what needs to be done, and add a TODO.

e.g.

TODO(ethkim): Fix the XML feed consumption to prevent breaking from XXXX

Reply:
That issue was covered by commit "Optimize each newspaper". We'll delete that line.